content/api/customer-alerts.apib

```diff
@@ -0,0 +1,314 @@
+      "test.com"
+    ]
+  }
+  ```
+
+## Alert Object
+
++ Data Structure: Attributes
+    + id (number) - Unique identifier of the alert.
+    + name (string) - Display name of the alert.
+    + sid (number) - Subaccount filter applied to the alert.
+    + any_sid (boolean) - Flag indicating all subaccounts should be evaluated for this alert.
+    + metric (enum) - Metric that the alert is configured to track.
+        + `"Monthly Sending Limit", "Health Score"`
+    + interval (enum) - Interval at which the alert is evaluated.
+        + `"24 Hours", "15 Minutes"`
+    + evaluation_enabled (boolean) - Flag indicating whether the alert should be evaluated.
+    + filters (array) - Filter objects to apply to the alert.
+    + threshold_evaluator (object) - Evaluator object for an alert with a trigger threshold.
+      + operator (enum) - Operator for alert evaluation to use.
+        + `"GT", "LT"`
+      + source (enum) - Source of value to compare against.
+        + `"Raw", "Day Over Day", "Week Over Week"`
+      + value (number) - Target value to compare against.
+    + email_channels (array) - Email addresses to send alert notifications to.
+
++ Sample
+
+  ```
+  {
+    "id": 117,
+    "name": "Alert Name Here",
+    "sid": 0,
+    "any_sid": false,
+    "metric": "Health Score",
+    "interval": "24 Hours",
+    "evaluation_enabled": true,
+    "filters": [
+      {
+        "filter_type": "Sending Domain", 
+        "filter_values": ["test.com"]
+      }
+    ],
+    "threshold_evaluator": {
+      "operator": "GT",
+      "source": "Raw",
+      "value": 0
+    },
+    "email_channels": [
+      "test@sparkpost.com",
+      "bigwig@test.com"
+    ]
+  }
+  ```
+
+### Create an Alert [POST /v1/customer-alerts]
+Creates a new alert
+
++ Data Structure
+  + name (string, required) - Display name of the alert.
+  + sid (number) - (Optional) Subaccount filter applied to the alert. This field must be set, or any_sid set to true.
+  + any_sid (boolean) - (Optional)  Flag indicating all subaccounts should be evaluated for this alert. If sid is not set, this must be true.
+  + metric (enum, required) - Metric that the alert is configured to track.
+      + `"Monthly Sending Limit", "Health Score"`
+  + interval (enum, required) - Interval at which the alert is evaluated.
+      + `"24 Hours", "15 Minutes"`
+  + evaluation_enabled (boolean, required) - Flag indicating whether the alert should be evaluated.
+  + filters (array, required) - Filter objects to apply to the alert.
+  + threshold_evaluator (object, required) - Evaluator object for an alert with a trigger threshold.
+    + operator (enum, required) - Operator for alert evaluation to use.
+      + `"GT", "LT"`
+    + source (enum, required) - Source of value to compare against.
+      + `"Raw", "Day Over Day", "Week Over Week"`
+    + value (number, required) - Target value to compare against.
+  + email_channels (array, required) - Email addresses to send alert notifications to.
```

Comment:
Should this be required for when we have future channels?

Reply:
Discussed and arrived at: should be in a larger channels object which is required, and at least one channel needs to be present in the object.